src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -802,12 +802,22 @@ bool FullyConnected::canBeExecutedInConv1x1() const {
         const auto& srcDims = srcMemPtr->getStaticDims();
         auto weightMemPtr = getParentEdgesAtPort(1)[0]->getMemoryPtr();
         const auto& weightDims = weightMemPtr->getStaticDims();
-        Dim M, N, K;
-        M = srcDims[inRank - 2];
+        // for original inner product semantics:
+        //  when input is 2D tensor
+        //    M in oneDNN will map to widthInBrg
+        //  when input is 3D tensor
+        //    M in oneDNN will map to widthInBrg*minibatch
+        // currently nwc mapping in brg:
```

Comment:
I think it is better to change `brg` to `conv` here and in the variable name, since the inner_product primitive also use brgemm algo.

Reply:
Updated.